### PR TITLE
Fix bytes2NativeString() in master, due to future.types.newbytes

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

File: master/buildbot/util/__init__.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def bytes2NativeString(x, encoding='utf-8'):
     @param encoding: an optional codec, default: 'utf-8'
     @return: a string of type C{str}
     """
-    if isinstance(x, bytes) and str != bytes:
+    if isinstance(x, bytes) and type("") != type(b""):
```
Comment (tardyp) [inline master/buildbot/util/__init__.py]: why dont you just use following?

    if isinstance(x, bytes) and PY3:

Comment (rodrigc) [inline master/buildbot/util/__init__.py]: I wanted to leave the future.utils constructs out of this function as much as possible,
and to future proof things, for example whenever Python 4 comes out.

Comment (tardyp) [inline master/buildbot/util/__init__.py]: well, at least we can add a comment saying that this means PY>=3

Comment (rodrigc) [inline master/buildbot/util/__init__.py]: OK I added a comment.

Comment (tardyp) [inline master/buildbot/util/__init__.py]: I meant a comment in the code. I dont see it. did you upload?

**Pull request merged**